src/pages/Home/components/PostFeedTab.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect } from 'react';
+import { toast } from 'react-toastify';
+import { useRecoilValue } from 'recoil';
+
+import PostPreviewCard from '../../Community/Feed/PostPreviewCard';
+import { IpostPreview } from '../../../common/types';
+import { userTokensOwnedAtom } from '../../../recoil/userInfo';
+import { userFinishedLoadingAtom } from '../../../recoil/appState';
+import { queryPostFeed } from '../../../common/services/Firebase/GetData/PostUtils';
+import LoadingDots from '../../../common/components/Util/LoadingDots';
+
+export default function PostFeedTab(): JSX.Element {
+  const ownedTokens = useRecoilValue(userTokensOwnedAtom);
+  const userFinishedLoading = useRecoilValue(userFinishedLoadingAtom);
+  const [loading, setLoading] = useState(true);
+  const [postList, setPostList] = useState<IpostPreview[]>([]);
+
+  async function getPostFeed() {
+    setLoading(true);
+    try {
+      const tokenOwnedSlice = Array.from(ownedTokens.keys()).slice(0, 10);
+      const queryPosts = await queryPostFeed(tokenOwnedSlice);
+      setPostList(queryPosts);
+      /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+    } catch (error: any) {
+      toast.error(error?.message);
```

Comment:
Let's make this human readable: "Failed to generate post feed".

Reply:
I need it for deployment, can explain